src/Utilities/FlowAnalysis/Framework/DataFlow/SetAbstractDomain.cs

```diff
@@ -23,7 +23,9 @@ public override int Compare(ImmutableHashSet<T> oldValue, ImmutableHashSet<T> ne
             }
 
             int result;
-            var isSubset = oldValue.IsSubsetOf(newValue);
+            // PERF: Avoid additional hash set allocation by using overload which takes
```

Comment:
nit: comment doesn't match code

Reply:
The new overload (extension method) used does take immutable hash set as the argument. Am I missing something?

Reply:
Oops, sorry, nevermind, didn't realize the existing `IsSubsetOf()` did instantiate a new HashSet.